rocketpy/environment/environment.py

```diff
@@ -468,8 +474,8 @@ def set_gravity_model(self, gravity):
         ----------
         gravity : None or Function source
```

Comment:
Something does not make sense here.

In line 290 you say:
```python
gravity : int, float, callable, string, array, optional
```

But here you mention that only None or Function source are allowed. 
Only one can be truth.

Can you provide more examples in the documentation please? A new user you never understand all the different ways that can be done to define the gravity.

Reply:
Fixed in [a2aa714](https://github.com/RocketPy-Team/RocketPy/pull/565/commits/a2aa71467b3be87c8124343cd49768c1ac4d6a9d).